uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/generator/AbstractLayoutGenerator.java

```diff
@@ -17,84 +17,94 @@
 
 import java.util.List;
 
-import com.google.gwt.user.client.ui.ComplexPanel;
 import com.google.gwt.user.client.ui.IsWidget;
-import com.google.gwt.user.client.ui.Panel;
-import org.gwtbootstrap3.client.ui.Column;
-import org.gwtbootstrap3.client.ui.Row;
+import com.google.gwt.user.client.ui.Widget;
+import org.jboss.errai.common.client.dom.DOMUtil;
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.common.client.ui.ElementWrapperWidget;
 import org.uberfire.ext.layout.editor.api.editor.LayoutColumn;
 import org.uberfire.ext.layout.editor.api.editor.LayoutComponent;
+import org.uberfire.ext.layout.editor.api.editor.LayoutInstance;
 import org.uberfire.ext.layout.editor.api.editor.LayoutRow;
 import org.uberfire.ext.layout.editor.api.editor.LayoutTemplate;
 import org.uberfire.ext.layout.editor.client.api.LayoutDragComponent;
 import org.uberfire.ext.layout.editor.client.api.RenderingContext;
-import org.uberfire.ext.layout.editor.client.infra.ColumnSizeBuilder;
 import org.uberfire.ext.layout.editor.client.infra.RowSizeBuilder;
 
 public abstract class AbstractLayoutGenerator implements LayoutGenerator {
 
     @Override
-    public Panel build(LayoutTemplate layoutTemplate) {
-        ComplexPanel container = getLayoutContainer();
+    public LayoutInstance build(LayoutTemplate layoutTemplate) {
+        HTMLElement container = createContainer(layoutTemplate);
```

Comment:
Can you please check if we can use elemental API's here?

Reply:
@ederign Can you please point me to an example?

Reply:
@dgutierr @adrielparedes is working on a new feature. @adrielparedes can you please give it an example for @dgutierr ?

@dgutierr next PR's we should all use elemental API's.